### PR TITLE
api/scheduler: add region and hot region schedule api

### PR DESCRIPTION
**Pull request by nolouch**
let we can dynamic add/remove it.

### CODE REVIEW

**Comment by siddontang** [comment]
any test to cover them?

**Comment by disksing** [comment]
LGTM.

**Comment by nolouch** [comment]
PTAL @siddontang

File: server/api/util.go [hunk 1]
```diff
@@ -53,3 +53,13 @@ func postJSON(cli *http.Client, url string, data []byte) error {
...
 	return nil
 }
+
+func doDelete(url string) error {
+	req, err := http.NewRequest("DELETE", url, nil)
+	if err != nil {
+		return err
+	}
+	res, err := http.DefaultClient.Do(req)
+	res.Body.Close()
```
Comment (siddontang) [inline server/api/util.go]: need check err then discard and close the body.

**Comment by siddontang** [comment]
Rest LGTM

**Comment by siddontang** [comment]
LGTM

**Pull request merged**